setup.py

```diff
@@ -69,15 +79,43 @@ def load_requirements(*requirements_paths):
         )
     return list(requirements)
 
+
+class VerifyVersionCommand(install):
```

Comment:
Let's do this changes(if needed) in the PR with the circleci-pypi integration

Reply:
Noted. Will remove these for now